pyproject.toml

```
@@ -1,14 +1,16 @@
 [tool.poetry]
 name = "pdfding"
-version = "0.1.1"
-description = ""
-authors = ["mrmn"]
+version = "1.4.1"
+description = "Selfhosted PDF manager, viewer and editor offering a seamless user experience on multiple devices."
+authors = ["mrmn2"]
 readme = "README.md"
+license = "AGPL-3.0-or-later"
+license-files = ["LICENSE"]
```

```suggestion
[project]
name = "pdfding"
version = "1.4.1"
description = "Selfhosted PDF manager, viewer and editor offering a seamless user experience on multiple devices."
authors = ["mrmn2"]
readme = "README.md"
license = "AGPL-3.0-only"
license-files = ["LICENSE"]
```

`license-files` does not work in the `tool.poetry` section. So I guess its best to just use `project` instead. Also PdfDing is licensed under AGPL-3.0 only so it is probably best to keep things consistent, what do you think?